Automattic/jetpack
# AI Assistant: Change conditions to show AI guidelines on input

Fixes #32773

## Proposed changes:

* Changes the conditions to show the guidelines. For the AI Assistant, it is now shown whenever there is any content in the container. For the Form extension it is now shown when there is a request happening or it is already done.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add an AI Assistant block
* Check that the guideline message is not shown at first
* Make a request
* Check that the guideline message appears only when there is some content.
* Add a Form
* Check that the guideline message is not shown at first either
* Make a request
* Check that the guideline message appears

It is easier to spot the difference by first testing on `trunk`.
